index.js

```diff
@@ -460,7 +460,7 @@ var ab2obj = function (ab) {
 
 // convert a hex string to an array buffer
 var hex2ab = function (s) {
-  return new Uint8Array(new Buffer(s, 'hex'))
+  return new Uint8Array(new Buffer(typeof s !== 'string' ? '' : s, 'hex'))
```

Comment:
in truth, it really isn't convenient to be throwing an error. however, returning `undefined` ought to work! your thoughts?